provider/storage_benchmark.py

```diff
@@ -155,6 +157,22 @@ def download_benchmark(self, mode, url, dst, timeout=300):
         self.session_cmd(self.download_cmds[mode].format(dst, url), timeout)
         self.env_files.append(dst)
 
+    def _get_route_ip(self):
+        """ Get the vm route ip address. """
+        route_iface = self.session_cmd("ip route | grep default | awk '{print $5}'").strip()
+        if not route_iface:
+            raise TestError('Failed to get the route table.')
+        return self.session_cmd(
+            "ifconfig %s | grep -E 'inet ' | awk '{print $2}'" % route_iface).strip()
+
+    def scp_benckmark(self, username, password, host_path, guest_path, port='22'):
+        """
+        Scp a benchmark tool from the local host to the guest.
+
+        """
+        scp_to_remote(self._get_route_ip(), port, username, password, host_path, guest_path)
```

Comment:
I am not very sure about the usage of this function,  only use `ssh` ?
If considering other type of transfer client, such as `rss` (for windows ?),  we can use `remote.copy_files_to`.

Reply:
just for linux, since the benchmark tools have been loaded in win_utils.iso, so don't need to copy them from host to guest.